assets/app/styles/_components.less

```diff
@@ -83,6 +83,7 @@
           overflow: hidden;
           text-overflow: ellipsis;
           line-height: 1.3;
+          white-space: nowrap;
```

Comment:
I still don't think you can just add nowrap here... we have long text in places around "triggered by a new image for [long name here]"